src/confluent_kafka/src/Consumer.c

```diff
@@ -1356,17 +1479,67 @@ static void Consumer_rebalance_cb (rd_kafka_t *rk, rd_kafka_resp_err_t err,
 		}
 	}
 
+        if (!strcmp(rd_kafka_rebalance_protocol(rk), "COOPERATIVE")) {
```

Comment:
Can return NULL (even though it shouldnt in this case, but better be careful, so assign to a temp var.

Reply:
ok